app/src/common/shared/org/mozilla/vrbrowser/telemetry/TelemetryWrapper.java

```diff
@@ -84,5 +93,41 @@ public static void stop() {
                 .scheduleUpload();
     }
 
+    @UiThread
+    public static void urlBarEvent(boolean aIsUrl) {
```

Comment:
nit: I think `aIsUrl` is pretty hard to read and parse mentally. Is that your coding style here? :)